surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+- I am a member of a Rust working group or project group.
+- I am a lead of a Rust working group or project group.
+- I am not a member of a Rust Team, working group, or project group, but I
+  would like to be.
+- I am not a member of a Rust team, working group, or project group, and I have
+  no intention to join one.
+
+> **justification**
+>
+> Cohort analysis. Some of the subsequent questions are only relevant to
+> certain cohorts.
+
+### What kind of work do you do within the Rust Project?
+
+Select all that apply:
+
+- contribution work
+- maintainership work
+- reviews
+- coordination
+- organization
+
+> **justification**
+>
+> Useful for cohort analysis. Problems will likely differ based on the work
+> being done. Those contributing new code will likely face diff problems than
+> those who mainly review or coordinate work.
+
+### What sort of work would you like to be focused on?
+
+Free Form.
+
+> **justification**
+>
+> Useful in identifying how satisified our contributors are with the work
+> they're currently doing or if we should be helping them branch out into new
+> areas / achieve new goals.
+>
+> This was one of the best received questions during the pre-survey interviews
+> and seemed to elicited thoughtful answers.
+
+### My experience contributing to the Rust Project so far has been positive.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### I feel positive about the project leadership and governance structure.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Suggested by @manishearth. Useful for identifying problems within project governance.
+
+### I feel positive about my own leadership within the project.
+
+**Skip if not a leader of a team or group**
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Suggested by @manishearth. Useful for identifying problems within project governance.
+
+### I feel my team/working group/project group is healthy.
+
+**Skip if not a member of the Rust Project.**
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### Are you burnt out or do you feel you're at risk of burning out?
+
+- I am burnt out.
+- I am at risk of burning out.
+- I am not burnt out but I have been in the past.
+- I have not been burnt out.
+
+> **justification**
+>
+> Answers a specific question we have.
+>
+> * Is burnout a major issue across the Rust Project?
+
+### What are the biggest issues you'd like to see fixed that are negatively impacting your ability to contribute to and/or maintain Rust?
```

Comment:
I wouldn't phrase this as the "biggest" issues if we're going to allow selecting as many options as possible. Either we limit to a certain number, we re-phrase the question to ask about which are issues (big or not), or we move to matrix model where for each option we ask how serious the issue appears to the respondent on some scale from "not an issue" to "significantly impairs my ability to contribute".

Reply:
of those options you presented the matrix model sounds most useful to me. I expect there may be issues that affect many people but with very low severity and if we only gather frequency and not amplitude we may get the wrong impression.